code/modules/shuttle/shuttle_manipulator.dm

```diff
@@ -9,9 +10,11 @@
 
 	icon = 'icons/obj/machines/shuttle_manipulator.dmi'
 	icon_state = "holograph_on"
-
+	/// Is this var used? Anywhere?
 	var/busy
-	// UI state variables
+	/// Used for cooldown, very obvious name, required due to shuttles spawning in the same location and causing the server to implode
+	var/cooldown_do_not_edit = 0
```

Comment:
if you don't want admins touching this var, you need to override the `vv_edit_var()` proc and early return on that var. Please change the var name to be more specific for what the cooldown is for.

Reply:
Applied, also added a warning explaining why you shouldn't varedit it, I can change that if you'd like